core/tests/ts-integration/src/env.ts

```diff
@@ -48,7 +48,7 @@ export async function waitForServer() {
  */
 export async function loadTestEnvironment(): Promise<TestEnvironment> {
     const network = process.env.CHAIN_ETH_NETWORK || 'localhost';
-    const nativeErc20Testing = process.env.NATIVE_ERC20_ADDRESS ? true : false; // if set, we assume user wants to test native erc20 tokens
+    const nativeErc20Testing = process.env.CONTRACTS_L1_NATIVE_ERC20_TOKEN_ADDR ? true : false; // if set, we assume user wants to test native erc20 tokens
 
     let mainWalletPK;
     if (nativeErc20Testing) {
```

Comment:
I know is not exactly part of this PR but you could do `if (process.env.CONTRACTS_L1_NATIVE_ERC20_TOKEN_ADDR)` and it should work fine.

Reply:
Yes it would, but I though that it would be better to declare a shorter variable once, and then use it three times, than calling calling `process.env.CONTRACTS_L1_NATIVE_ERC20_TOKEN_ADDR` each time.